edx_solutions_api_integration/utils.py

```diff
@@ -29,6 +29,10 @@
 COURSE_METRICS_CACHE_TTL = 30 * 60
 
 
+PERMISSION_GROUPS = {
```

Comment:
We can discuss about the possible placement of this dict as well. There may be a better location for this thing.